tools/libxl/libxl_vkb.c

```diff
@@ -28,6 +28,18 @@ static int libxl__device_vkb_dm_needed(libxl_device_vkb *vkb, uint32_t domid)
     return 0;
 }
 
+static int libxl__set_xenstore_vkb(libxl__gc *gc, uint32_t domid,
+                                   libxl_device_vkb *vkb,
+                                   flexarray_t *back, flexarray_t *front,
+                                   flexarray_t *ro_front)
+{
+    if (vkb->id) {
```

Comment:
no brackets around single expression?

Reply:
Every indented statement is braced, but blocks that contain just one
statement may have the braces omitted.